### PR TITLE
feat(execution-plan): refine execution plan

### PR DESCRIPTION
**Pull request by baurine**
## What Did

- Show the execution plan text same as the `explain analyze` result.
- Show the execution plan as table format
- Adjust the different formats order
- Make execution plan text always expand

### Before

image

### After

image

image

### CODE REVIEW

File: pkg/apiserver/slowquery/queries.go [hunk 1]
```diff
@@ -109,11 +109,19 @@ func QuerySlowLogList(req *GetListRequest, sysSchema *utils.SysSchema, db *gorm.
...
 		Where("Conn_id = ?", req.ConnectID).
 		First(&result).Error
+	if err != nil {
+		err = db.
+			Select("*, (UNIX_TIMESTAMP(Time) + 0E0) AS timestamp").
+			Where("Digest = ?", req.Digest).
+			Where("Time = FROM_UNIXTIME(?)", req.Timestamp).
+			Where("Conn_id = ?", req.ConnectID).
+			First(&result).Error
+	}
```
Comment (mornyx) [inline pkg/apiserver/slowquery/queries.go:124]: Is it to eliminate any errors generated by `tidb_decode_binary_plan`? What errors does it generate?

Comment (baurine) [reply pkg/apiserver/slowquery/queries.go:124]: This method may not exist in the old tidb version, so it's for compatibility.

File: ui/packages/tidb-dashboard-lib/src/apps/Statement/pages/Detail/PlanDetail.tsx [hunk 2]
```diff
@@ -84,8 +85,8 @@ function PlanDetail({ query }: IPlanDetailProps) {
     setDetailExpand((prev) => ({ ...prev, prev_query: !prev.prev_query }))
   const toggleQuery = () =>
     setDetailExpand((prev) => ({ ...prev, query: !prev.query }))
-  const togglePlan = () =>
-    setDetailExpand((prev) => ({ ...prev, plan: !prev.plan }))
+  // const togglePlan = () =>
+  //   setDetailExpand((prev) => ({ ...prev, plan: !prev.plan }))
```
Comment (shhdgit) [inline ui/packages/tidb-dashboard-lib/src/apps/Statement/pages/Detail/PlanDetail.tsx:89]: Why don't we need collapse the plan anymore?

Comment (baurine) [reply ui/packages/tidb-dashboard-lib/src/apps/Statement/pages/Detail/PlanDetail.tsx:89]: Communicated with the DBA, they don't need it, they want it be expanded at default, so they can see the full content as quickly as possible.

**Comment by mornyx** [comment]
Go part LTGM

**Review by mornyx — APPROVED** [review]

**Review by shhdgit — APPROVED** [review]

**Pull request merged**